src/test/fuzz/process_message.cpp

```diff
@@ -55,21 +55,25 @@ void initialize_process_message()
 void fuzz_target(const std::vector<uint8_t>& buffer, const std::string& LIMIT_TO_MESSAGE_TYPE)
 {
     FuzzedDataProvider fuzzed_data_provider(buffer.data(), buffer.size());
+
     ConnmanTestMsg& connman = *(ConnmanTestMsg*)g_setup->m_node.connman.get();
     TestChainState& chainstate = *(TestChainState*)&g_setup->m_node.chainman->ActiveChainstate();
+    SetMockTime(1610000000); // any time to successfully reset ibd
     chainstate.ResetIbd();
+
     const std::string random_message_type{fuzzed_data_provider.ConsumeBytesAsString(CMessageHeader::COMMAND_SIZE).c_str()};
     if (!LIMIT_TO_MESSAGE_TYPE.empty() && random_message_type != LIMIT_TO_MESSAGE_TYPE) {
         return;
     }
-    const bool jump_out_of_ibd{fuzzed_data_provider.ConsumeBool()};
-    if (jump_out_of_ibd) chainstate.JumpOutOfIbd();
```

Comment:
Now that `TestChainState::JumpOutOfIbd` is unused we can drop it?

Reply:
I think there are some other pull requests using it, so I'll keep it for now

Reply:
https://github.com/bitcoin/bitcoin/pull/20196/files#r556430946

Reply:
Makes sense. Thanks for clarifying!